tests/functional/api/test_import_export.py

```diff
@@ -78,3 +78,23 @@ def test_project_remote_import(gl):
         "File url is blocked: Only allowed schemes are https"
         in err_info.value.error_message
     )
+
+
+def test_project_remote_import_s3(gl):
+    gl.features.set("import_project_from_remote_file_s3", True)
+    with pytest.raises(gitlab.exceptions.GitlabImportError) as err_info:
+        gl.projects.remote_import_s3(
+            "remote-project",
+            "aws-region",
+            "aws-bucket-name",
+            "aws-file-key",
+            "aws-access-key-id",
+            "secret-access-key",
+            "remote-project",
+            "root",
+        )
```

Comment:
Something I just realized here is that this will trigger real requests (from the docker-compose GitLab instance I assume) to the actual AWS API. This is something we probably don't want to have long-term. I don't want to block this PR though but if you have any ideas, for example by blocking networking in a certain way in our docker-compose setup, that might be a good follow-up

Reply:
Hmm it will not probably be a simple config change, will have to update the exiswting s3 test case too as the error response might change now since we never go to s3

Reply:
Ah true. It would be better if GitLab supported other instances such as minio, then we would maybe just configure that. Not urgent then at all, thanks!

Reply:
Ok. Will keep this in my mind. Will contribute if I find something!

Reply:
I see that the Ruby client supports an `endpoint` param, this would probably work, but needs an upstream contribution first.

https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/S3/Client.html

Reply:
They do have configurable endpoints for other scopes https://gitlab.com/gitlab-org/omnibus-gitlab/blob/master/files/gitlab-config-template/gitlab.rb.template?_gl=1%2a1vxyfkb%2a_ga%2aMzg2MTk2NTY4LjE2NTk4OTgxMzU.%2a_ga_ENFH3X7M5Y%2aMTY2NzMzMDYyNC4xMS4xLjE2NjczMzE5MjguMC4wLjA.#L379.

Reply:
but this might require the param at api level,since the client is probably generated at runtime for each api call